### ISSUE TITLE
Documentation of OPTIONS is poor

### ISSUE THREAD

**Issue by nacesprin** [pawelczak/EasyAutocomplete#341]
Hello. This is a GREAT plugin, but the documentation related to OPTIONS is very poor. I must to explore de github code to learn and discover all Options like minCharNumber and adjustWidth, among others.

Should it be great to get documentation about the plugin options.

Thanks.

**Comment by adarshmadrecha** [comment]
I completely agree. Dear Author of the plugin, please update the plugin and also the documentation.

**Comment by schroef** [comment]
I would like to know more about the custom input. I cant find how to make this list match, it seems match only works on the precomposed onces.

**Comment by schroef** [comment]
I got it working now, needed todo quite a lot of testing. But its really nice actually.

www.promotiespullen.com

**Comment by LiamDawe** [comment]
Any update on this? I wouldn't say it's outright terrible, it's enough to get you going.

However, it does completely miss telling you stuff like "adjustWidth", setting it to false stops the plugin forcing a width on the input box (which seems to be quite pointless).

**Comment by LiamDawe** [comment]
I am well aware I can change CSS, you're completely missing the point.

Options should be documented, not hidden.

**Comment by LiamDawe** [comment]
No I decided to completely ignore the website...

Of course I have checked it. As I'm sure others have. The entire reason this ticket exists, is because options aren't listed.
